qcodes/instrument_drivers/Gentec/Gentec_Maestro.py

```diff
@@ -0,0 +1,66 @@
+from qcodes import VisaInstrument
+from qcodes.utils.helpers import create_on_off_val_mapping
+
+
+class Gentec_Maestro(VisaInstrument):
+    """
+    Instrument driver for the Gentec Maestro powermeter.
+    Args:
+        name (str): Instrument name.
+        address (str): ASRL address
+        baud_rate (int): Baud rate for the connection.
+    Attributes:
+        model (str): Model identification.
+        firmware_version (str): Firmware version.
+        
+    """
+
+    def __init__(self, name, address, baud_rate=115200, **kwargs):
+
+        super().__init__(name, address, **kwargs)
+
+        # set baud rate
+        self.visa_handle.baud_rate = baud_rate
+
+        # get instrument information
+        self.model = self.visa_handle.query('*VER').split()[0]
+        self.firmware_version = self.visa_handle.query('*VER').split()[2]
```

Comment:
Please make a spearate (private?) method for this query `self.visa_handle.query('*VER')` and re-use it here. (perhaps, smth like `model, _, fw_version = self._query_versions()`)